es-rest-client-1.3/es-client-java-jersey/src/test/java/com/bazaarvoice/elasticsearch/client/GetTest.java

```diff
@@ -47,4 +48,14 @@
         assertTrue(get.getFields().isEmpty());
         assertEquals(get.getSource().get("field"), "value");
     }
+
+    @Test(expectedExceptions = IndexMissingException.class)
```

Comment:
I generally prefer not to test this way. I like this better:

```
final GetRequestBuilder getRequestBuilder = restClient().prepareGet(index + "nosuch", type, id);
final ListenableActionFuture<GetResponse> execute1 = getRequestBuilder.execute();
try {
    execute1.actionGet();
    fail("Expected an IndexMissingException");
} catch (IndexMissingException e) {
    assertEquals(e.index(), index+"nosuch");
}
```

You get more control over exactly what you're testing without resorting to spending a lot of time reading through testng's manual to figure out how to use their matchers, etc.